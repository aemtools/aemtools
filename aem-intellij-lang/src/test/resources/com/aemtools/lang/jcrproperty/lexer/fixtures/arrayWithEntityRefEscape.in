[&quot;Test &amp; Test&quot;,&apos;Test2 &amp; Test2&apos;]
