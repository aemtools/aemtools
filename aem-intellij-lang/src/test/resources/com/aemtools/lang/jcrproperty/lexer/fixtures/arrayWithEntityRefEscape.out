[ ('[')
VALID_XML_ENTITY_REF_ESCAPE_TOKEN ('&quot;')
ARRAY_VALUE_TOKEN ('Test ')
VALID_XML_ENTITY_REF_ESCAPE_TOKEN ('&amp;')
ARRAY_VALUE_TOKEN (' Test')
VALID_XML_ENTITY_REF_ESCAPE_TOKEN ('&quot;')
, (',')
VALID_XML_ENTITY_REF_ESCAPE_TOKEN ('&apos;')
ARRAY_VALUE_TOKEN ('Test2 ')
VALID_XML_ENTITY_REF_ESCAPE_TOKEN ('&amp;')
ARRAY_VALUE_TOKEN (' Test2')
VALID_XML_ENTITY_REF_ESCAPE_TOKEN ('&apos;')
] (']')
